va/caa_test.go

```diff
@@ -103,6 +104,18 @@ func (mock caaMockDNS) LookupCAA(_ context.Context, domain string) ([]*dns.CAA,
 		record.Tag = "issue"
 		record.Value = "letsencrypt.org; validationmethods=http-01,dns-01"
 		results = append(results, &record)
+	case "present-dns-only-correct-accounturi.com":
```

Comment:
Can you also add cases for invalid parameter tags and values.

Reply:
Will do, as soon as I have clarification on the expected behaviour.

Reply:
Added.